### ISSUE TITLE
requireNativeComponent "RNGADNativeView" was not found in the UIManager erro

### ISSUE THREAD

**Issue by olkunmustafa** [ammarahm-ed/react-native-admob-native-ads#61]
Whenever I tried to use this library throws error below
"**requireNativeComponent "RNGADNativeView" was not found in the UIManager**"

The versions are like below
"react": "16.8.3",
"react-native": "0.59.9",
"react-native-admob": "^2.0.0-beta.6",
"react-native-admob-native-ads": "^0.3.5",
"react-native-gesture-handler": "1.6.1"

I am not using expo. That is bare react native project.

![Simulator Screen Shot - iPhone X - 2020-08-16 at 15 19 01](https://user-images.githubusercontent.com/5277158/90334118-def65300-dfd3-11ea-83ed-bfd2d159224c.png)

**Comment by ammarahm-ed** [comment]
Link the project manually.

**Comment by pk936** [comment]
I tried react-native link react-native-admob-native-ads. But it did not work.

**Comment by devGuerra** [comment]
me too

**Comment by cactisolutions** [comment]
@devGuerra same, tried react-native link react-native-admob-native-ads. But it did not work, I also tried to link manually but that did not work either.

**Comment by salp13** [comment]
@devGuerra @cactisolutions  I am also having this issue, did anyone find a solution / the problem?

**Comment by adamsadiq7** [comment]
any solution?

**Comment by DavidAPears** [comment]
Also facing this

**Comment by mhr996** [comment]
Same issue, anybody found a solution?

**Comment by ammarahm-ed** [comment]
@mhr996 are you using expo?

**Comment by mhr996** [comment]
@ammarahm-ed Yes, but I'm using a development build, so I'm not limited to Expo Go features

**Comment by muriloalvesdev** [comment]
@ammarahm-ed 
using expo yes you closed but not published a resolution before..

**Comment by muriloalvesdev** [comment]
And they even opened another issue about
https://github.com/ammarahm-ed/react-native-admob-native-ads/issues/336

**Comment by ammarahm-ed** [comment]
Expo support added in v0.6.3

**Comment by okinawaa** [comment]
I facing same issue! 
<img width="632" alt="image" src="https://github.com/user-attachments/assets/48acfe97-838a-4c56-a1da-88c55a104b5e">
